cpp/src/copying/slice.cpp

```diff
@@ -22,6 +22,8 @@
 #include <cudf/utilities/error.hpp>
 
 #include <algorithm>
+#include "cudf_test/column_wrapper.hpp"
+#include "thrust/iterator/transform_iterator.h"
```

Comment:
Let's not depend on cudf_test (add a header). And use angle brackets and group headers.

```suggestion
#include <cudf_test/column_wrapper.hpp>

#include <thrust/iterator/transform_iterator.h>
```

Reply:
is there a vs code setting to use angle brackets?